source/connect/connection-options.txt

```diff
@@ -106,6 +106,26 @@ Network Compression
        | **MongoClient Example**: ``zlibCompressionLevel = 3``
        | **Connection URI Example**: ``zlibCompressionLevel=3``
 
+Timeouts
+~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Connection Option
+     - Description
+
+   * - **timeoutMS**
+     - | The number of milliseconds each call to the server must complete within. If an
```

Comment:
I still think this should say "driver operation" not "call to the server".

Since one driver operation can run multiple server operations (eg bulk write), timeoutMS can span multiple server operations.

Reply:
My hesitation was that "driver operation" can mean something other than requesting that MongoDB perform one or more operations, right? I think we want to specifically identify the subset of PyMongo operations that involve communicating with the server. If you think that === "driver operation", then I'm happy to use that language!

Reply:
I think "driver operation" is more clear. Here's how the spec describes timeoutMS:
> This specification outlines a new timeoutMS option to govern the amount of time that a single operation can execute before control is returned to the user. This timeout applies to all of the work done to execute the operation, including but not limited to server selection, connection checkout, and server-side execution.